timemachine/testsystems/relative.py

```diff
@@ -3,13 +3,14 @@
 import numpy as np
 from rdkit import Chem
 
+import timemachine
 from timemachine.fe import free_energy, topology
 from timemachine.ff import Forcefield
 from timemachine.ff.handlers.deserialize import deserialize_handlers
 
 from pathlib import Path
 
-root = Path(__file__).parent.parent
+root = Path(timemachine.__file__).parent.parent
```

Comment:
huh - didn't know you could do __file__ on a module... does this point to the __init__.py or something?

Reply:
Yep -- exactly!

This way is currently used in a couple other places ([tests/common.py](https://github.com/proteneer/timemachine/blob/a55a509a9db94b346b7ad75b02514ededd87e119/tests/common.py#L34), [RABFE script](https://github.com/proteneer/timemachine/blob/a55a509a9db94b346b7ad75b02514ededd87e119/examples/validate_relative_binding.py#L203)) -- avoiding unnecessary path search is [intended later](https://github.com/proteneer/timemachine/issues/602).